packages/protocol/contracts/common/GoldWhitelist.sol

```diff
@@ -0,0 +1,73 @@
+pragma solidity ^0.5.3;
+
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "./UsingRegistry.sol";
+
+/**
+ * @title Holds a whitelist of addresses for which Gold transfers cannot be frozen.
```

Comment:
`Holds a whitelist of addresses for which transfers should not be frozen so that network initialization can take place`